eb

```diff
@@ -121,8 +124,16 @@ export EASYBUILD_HOOKS=$EASYBUILD_CONFIG_ROOT/cc_hooks.py
 export EASYBUILD_BUILDPATH=$EASYBUILD_BUILDPATH/$RSNT_ARCH
 export LANG=en_US.UTF-8
 export LC_ALL=en_US.UTF-8
-if [ ! -d /cvmfs/local ]; then
-	export EASYBUILD_USE_CCACHE=''
+if [[ $HOSTNAME =~ archimedes.c3.ca$ && "$RSNT_ARCH" == "avx2" ]]; then
+	EASYBUILD_USE_CCACHE=/tmp/ccache
```

Comment:
why just for avx2 ? we could have a `/tmp/ccache/$RSNT_ARCH` ?

Reply:
(note that we've always disabled ccache for avx512 builds)

Reply:
ok